test/jobs/queue_transcoded_items_for_codec_conversion_job_test.rb

```diff
@@ -0,0 +1,4 @@
+require 'test_helper'
+
+class QueueTranscodedItemsForCodecConversionJobTest < ActiveJob::TestCase
+end
```

Comment:
This is still missing tests.

I would test:
* Are the files deleted?
* Are the expected amount of jobs created?